### PR TITLE
Storyteller play button style

### PR DESCRIPTION
**Pull request by jsavella**
Update playbutton from square to rounded style 

Fix #<gh-issue-id>

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri-eds--esri.aem.live/
- After: https://storytellerPlayButton--esri-eds--esri.aem.live/

playbutton-square

round-play

### CODE REVIEW

File: eds/blocks/storyteller/storyteller.css [hunk 1]
```diff
@@ -186,11 +183,66 @@
...
+  padding: 0 14px;
+  border-radius: 50%;
+  align-items: center;
+  justify-content: center;
+  background-color: transparent;
+}
+
+.storyteller .foreground-container .video-container .video-playbutton.paused:after {
+  mask: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMCAyMCI+PHBhdGggZD0iTTUgMTUuMjVWNC43N2ExLjQ0IDEuNDQgMCAwMTEuNDQtMS42MiAxLjg2IDEuODYgMCAwMTEuMTEuMzFsOC41MyA1Yy43Ni40NCAxLjE3LjggMS4xNyAxLjUxcy0uNDEgMS4wNy0xLjE3IDEuNTFsLTguNTMgNWExLjg2IDEuODYgMCAwMS0xLjExLjMxQTEuNDIgMS40MiAwIDAxNSAxNS4yNXoiLz48L3N2Zz4=");
+  -webkit-mask: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMCAyMCI+PHBhdGggZD0iTTUgMTUuMjVWNC43N2ExLjQ0IDEuNDQgMCAwMTEuNDQtMS42MiAxLjg2IDEuODYgMCAwMTEuMTEuMzFsOC41MyA1Yy43Ni40NCAxLjE3LjggMS4xNyAxLjUxcy0uNDEgMS4wNy0xLjE3IDEuNTFsLTguNTMgNWExLjg2IDEuODYgMCAwMS0xLjExLjMxQTEuNDIgMS40MiAwIDAxNSAxNS4yNXoiLz48L3N2Zz4=")
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:215]: don't need vendor prefix

File: eds/blocks/storyteller/storyteller.css [hunk 2]
```diff
@@ -186,11 +183,66 @@
...
+}
+
+.storyteller .foreground-container .video-container .video-playbutton:after {
+  content: "";
+  mask-size: cover;
+  -webkit-mask-size: cover;
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:221]: dont need

File: eds/blocks/storyteller/storyteller.css [hunk 3]
```diff
@@ -186,11 +183,66 @@
...
+  display: inline-block;
+  inline-size: 20px;
+  block-size: 20px;
+  position: absolute;
+  background-size: 20px 20px;
+  background-color: #ffffff;
+  -webkit-mask: url("data:image/svg+xml;base64,PHN2ZyBpZD0iRXhwb3J0YWJsZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2aWV3Qm94PSIwIDAgMjAgMjAiPjxkZWZzPjxzdHlsZT4uY2xzLTF7ZmlsbDojMWQxZDFmfTwvc3R5bGU+PC9kZWZzPjxnIGlkPSJwYXVzZSI+PHJlY3QgY2xhc3M9ImNscy0xIiB4PSIzLjc1IiB5PSIzIiB3aWR0aD0iNC41IiBoZWlnaHQ9IjE0IiByeD0iMS41Ii8+PHJlY3QgY2xhc3M9ImNscy0xIiB4PSIxMS43NSIgeT0iMyIgd2lkdGg9IjQuNSIgaGVpZ2h0PSIxNCIgcng9IjEuNSIvPjwvZz48L3N2Zz4=");
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:228]: same

File: eds/blocks/storyteller/storyteller.css [hunk 4]
```diff
@@ -331,7 +387,7 @@
   }
   
   .storyteller button.video-playbutton {
-    margin-inline-end: var(--space-4); 
+    /* margin-inline-end: var(--space-4);  */
   }
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:391]: remove commented code

**Review by twhite313 — CHANGES_REQUESTED** [review]

File: eds/blocks/storyteller/storyteller.css (hunk 4 above)
Comment (jsavella) [reply eds/blocks/storyteller/storyteller.css:385]: done

**Review by twhite313 — APPROVED** [review]

**Pull request merged**